charts/app-config/values-production.yaml

```diff
@@ -190,7 +190,7 @@ govukApplications:
     appResources:
       limits:
         cpu: 4
-        memory: 1500Mi
+        memory: 2000Mi
       requests:
         cpu: 2
         memory: 1Gi
```

Comment:
Might wanna increase the request as well. Maybe 1500?

Reply:
thats sensible done